cumulus_etl/chart_review/cli.py

```diff
@@ -8,7 +8,7 @@
 import ctakesclient
 from ctakesclient.typesystem import Polarity
 
-from cumulus_etl import cli_utils, common, deid, errors, fhir_client, fhir_common, loaders, nlp, store
+from cumulus_etl import cli_utils, common, deid, errors, fhir, loaders, nlp, store
```

Comment:
You're going to see a lot of light refactoring. I was adding a new "FHIR spec"-like bit of code, so I took the opportunity to gather the FHIR code into one subdirectory.

`fhir_client` -> `fhir`
`fhir_common` -> `fhir`
(new) `fhir_schemas` -> `fhir`

And I moved the general purpose `FatalError` out of `fhir_client` and into `errors`.

Reply:
this generally feels like a very good move to me.